compute/address/address_test.go

```diff
@@ -262,3 +262,111 @@ func TestReserveNewGlobalExternal(t *testing.T) {
 		t.Errorf("Global address %v not found in the list of reserved addresses", addressName)
 	}
 }
+
+func TestReleaseGlobalExternal(t *testing.T) {
+	ctx := context.Background()
+	var seededRand = rand.New(
+		rand.NewSource(time.Now().UnixNano()))
+	tc := testutil.SystemTest(t)
+	addressName := "test-global-address-" + fmt.Sprint(seededRand.Int())
+
+	buf := &bytes.Buffer{}
+
+	defer func() {
+		if err := deleteIPAddress(ctx, tc.ProjectID, "", addressName); err != nil {
+			// cleanup in case of unexpected result
+		}
+	}()
+
+	_, err := reserveNewGlobalExternal(buf, tc.ProjectID, addressName, true)
+	if err != nil {
+		t.Errorf("reserveNewGlobalExternal got err: %v", err)
+	}
```

Comment:
```suggestion
	}
	buf.Reset()
```
you should reset your buffer between steps here, otherwise it will contain *all* the output from previous steps. (at minimum, reset before you do the release, which is what we're testing here).

Reply:
done and rebased on fresh main 👍